### PR TITLE
Sd on host integrations 7

### PR DESCRIPTION
**Pull request by homelessbirds**
NR-139317

### CODE REVIEW

File: src/content/docs/browser/browser-integrations/mexn-integration.mdx [hunk 1]
```diff
@@ -24,41 +24,51 @@ After setting up our MExN integration, we give you a dashboard for your MExN web
...
+###  Install MongoDB quickstart
 
 1. Check out our MongoDB agent requirements in our [documentation](/docs/infrastructure/host-integrations/host-integrations-list/mongodb-monitoring-integration/)  before installing the agent.
 2. Open the [MongoDB quickstart installation](https://newrelic.com/instant-observability/mongodb). 
 3. Click  **Install now** to start the MongoDB agent installation.
+</Step>
 
-### Step 3: Install Node.js agent
+<Step>
+###  Install Node.js agent
```
Comment (zuluecho9) [inline src/content/docs/browser/browser-integrations/mexn-integration.mdx:45]: just a note that we're lacking a lot of anchor links @homelessbirds

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-hadoop-monitoring-integration.mdx [hunk 2]
```diff
@@ -0,0 +1,146 @@
+---
+title: Apache Hadoop integration
```
Comment (zuluecho9) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-hadoop-monitoring-integration.mdx:2]: i think our general pattern is that we like 'monitoring' in the titles of these, for SEO value

**Review by nbaenam — APPROVED** [review]
I've committed my modifications because there were a lot, hope you don't mind.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/wordpress-fullstack-integration.mdx [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+1.  Create a file named `read-wordpress-files-config.yml` in this path:
+
+  ```shell
+  /etc/newrelic-infra/integrations.d
+  ```
+
+2.  Update the `read-wordpress-files-config.yml` with these details:
+
+    * `INSERT_EVENT_TYPE`. An `event_type` is a New Relic database table that you can query using NRQL. An example of `event_type` would be  `WPDirectories/`
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/wordpress-fullstack-integration.mdx:76]:
```suggestion
    * `INSERT_EVENT_TYPE`. An `event_type` is a New Relic database table that you can query using NRQL. An example of `event_type` would be  `WPDirectories`
```

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/postfix-monitoring-integration.mdx [hunk 4]
```diff
@@ -0,0 +1,406 @@
...
+
+    #Read the message bounce details.              
+        - event_type: PostfixBounceMessage
+          commands:    
+            - run: cat /var/log/mail.log | pflogsumm | grep -A 10 'message bounce detail (by relay)' | sed '/message reject detail/q'
+              split: horizontal
+              header_split_by: \s+(\w+)\s+(.*)
+              set_header: [Bounce.msgCount, bounceMessage]
+              regex_match: true
+              split_by: \s*(\d+)\s\s\s(.*.)
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/postfix-monitoring-integration.mdx:325]: Indentation was perfect until line 140.

Need to align the indentation of lines 143 to 325 with line 134

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/postfix-monitoring-integration.mdx [hunk 5]
```diff
@@ -0,0 +1,406 @@
...
+On Linux machines, your log file named `logging.yml` should be present in this path:
+
+```shell
+/etc/newrelic-infra/logging.d/
+```
+
+After creating the log file, add the following script to the `logging.yml` file:
+
+```yml
+  - name: mail.log
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/postfix-monitoring-integration.mdx:347]:
```suggestion
logs:
  - name: mail.log
```

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/sendmail-monitoring-integration.mdx [hunk 6]
```diff
@@ -0,0 +1,260 @@
...
+              header_split_by: \s+(\d+)\s+(.*.)
+              set_header: [hostCount.total, hostname.total]
+              regex_match: true
+              split_by: \s+(\d+)\s+(.*.)
+            - run: cat /var/log/mail.log | grep "from=<.*.>" | grep "$(date +'%b %e')" | awk '{print $7}' | grep -oE '[^@]+[a-zA-Z0-9\-\.]+\.[a-zA-Z]{2,}' | sort | uniq -c
+              split: horizontal
+              header_split_by: \s+(\d+)\s+(.*.)
+              set_header: [hostCount.latest, hostname.latest]
+              regex_match: true
+              split_by: \s+(\d+)\s+(.*.)
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/sendmail-monitoring-integration.mdx:178]: Indentation was perfect until line 94.

Need to align the indentation of lines 97 to 165 with line 91.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/sendmail-monitoring-integration.mdx [hunk 7]
```diff
@@ -0,0 +1,260 @@
...
+
+```shell
+/etc/newrelic-infra/logging.d/
+```
+
+After creating the log file, add the following script to the `logging.yml` file:
+
+```yml
+  
+  - name: mail.log
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/sendmail-monitoring-integration.mdx:198]:
```suggestion
logs:
  - name: mail.log
```

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/sonarqube-monitoring-integration.mdx [hunk 8]
```diff
@@ -0,0 +1,189 @@
...
+3. Update these fields:
+
+    * cluster_name: `YOUR_DESIRED_CLUSTER_NAME`
+    * bearer_token_file: `USER_TOKEN_FILE_PATH`
+    * urls: `[“https://YOUR_HOST_IP:9000/api/monitoring/metrics”]`
+    * Insecure_skip_verify: `true`
+
+Your `nri-prometheus-config.yml` file should look like this:
+
+```yml
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/sonarqube-monitoring-integration.mdx:72]: please use the below reference link to cross check the indentation of this yml.
https://github.com/newrelic/nri-prometheus/blob/main/configs/nri-prometheus-config.yml.sample
Thank you 👍

**Review by pkosanam — CHANGES_REQUESTED** [review]

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/sendmail-monitoring-integration.mdx (hunk 6 above)
Comment (homelessbirds) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/sendmail-monitoring-integration.mdx:178]: Done

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/postfix-monitoring-integration.mdx (hunk 4 above)
Comment (homelessbirds) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/postfix-monitoring-integration.mdx:325]: done

**Pull request merged**